packages/mercury/docs/installation-for-angular.md

```diff
@@ -0,0 +1,344 @@
+<img src="./images/mercury-brand.svg" alt="Mercury brand" width="30" />
+
+Mercury Design System
+
+# <img src="./images/angular.svg" alt="Angular logo" width="30" /> &nbsp;Installation for Angular
+
+- [1. Usage](#1-usage)
+
+- [2. Installation](#2-installation)
+
+  - [2.1. Decide the assets paths](#21-decide-the-assets-paths)
+
+  - [2.2. Install](#22-install)
+
+- [3. Setup](#3-setup)
+
+  - [3.1 Import declarations](#31-import-declarations)
+
+  - [3.2 Include Icons](#32-include-icons)
+
+  - [3.3 Style the base application](#33-style-the-base-application)
+
+  - [3.4 Style the components with the bundles](#34-style-the-components-with-the-bundles)
+
+- [4. Copy bundles and assets](#4-copy-bundles-and-assets)
+
+- [Read additional documentation](#read-additional-documentation)
+
+## 1. Usage
+
+This repository provides the following assets to implement the **Mercury DS** on top of the **Chameleon Controls Library**:
+
+- CSS bundles to style the [Chameleon](https://github.com/genexuslabs/chameleon-controls-library) white label components.
+- Pre-packaged font assets (Inter family).
+- Consistent icon set for UI/UX.
+
+The CSS is organized into bundles — independent `.css` files that style specific components. For example, the `components/button` bundle styles buttons, the `components/checkbox` bundle styles the ch-checkbox component, and so on.
+
+## 2. Installation
+
+### 2.1. Decide the assets paths
+
+First of all, you must decide the paths where the CSS bundles, font assets, and icon set will be contained in the `dist` folder of your final application.
+
+We will refer to those paths with the following names:
+
+| Reference                      | Meaning                                                                                | Example                   |
+| ------------------------------ | -------------------------------------------------------------------------------------- | ------------------------- |
+| `{{ CSS outDir path }}`        | An untracked folder for generating the CSS of the application.                         | `src/assets/mercury-css/` |
+| `{{ CSS bundles final path }}` | Path in the final application (`dist` folder) where the CSS bundles will be consumed.  | `/assets/css/`            |
+| `{{ Fonts final path }}`       | Path in the final application (`dist` folder) where the custom fonts will be consumed. | `/assets/fonts/`          |
+| `{{ Icons final path }}`       | Path in the final application (`dist` folder) where the icons will be consumed.        | `/assets/icons/`          |
+
+Update your `package.json` with the following scripts, and dependencies:
+
+```json
+  "scripts": {
+    "dev": "npm run build.scss && ...",
+    "start": "npm run build.scss && ...",
+    "build": "npm run build.scss && ...",
+    "build.scss": "mercury --i={{ Icons final path }} --f={{ Fonts path final }} --outDir={{ CSS outDir path }}"
```

Comment:
We no longer distribute scss and in the previous section we used the words "CSS bundles".

```suggestion
    "dev": "npm run build.css && ...",
    "start": "npm run build.css && ...",
    "build": "npm run build.css && ...",
    "build.css": "mercury --i={{ Icons final path }} --f={{ Fonts path final }} --outDir={{ CSS outDir path }}"
```